xml_file_generation/generate_xml_datm.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+# Copyright 2023 ACCESS-NRI and contributors. See the top-level COPYRIGHT file for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Contact: Ezhilsabareesh Kannadasan
```

Comment:
Could you add a sentence here describing what this script is for?

Reply:
Thanks Dougie, added in this commit [48836d1](https://github.com/COSIMA/om3-scripts/commit/48836d1dd84e340b9afcc70f655e58799e3f0b48)